docs/concepts.rst

```diff
@@ -5,9 +5,9 @@ The Product Delivery Model
 -----------------------------
```

Comment:
**Key Concepts**

Key concept. Developer jargon. 10-cent words that sound like a technical writer patting himself on the back. (Hi there!) We imagine your cursor subconsciously drifting its way to the hyperlink for the next chapter already. BUT in an effort to give you the tools to build and test customized features with CumulusCI, it’s only right that we demystify the process by explaining the _how_ and the _why_ in order to help you achieve your goal of seamless automation.

Don’t worry, we’ll make this as painless as possible.